mutiny-core/Cargo.toml

```diff
@@ -20,15 +20,14 @@ cfg-if = "1.0.0"
 wasm-bindgen = "0.2.84"
 bip39 = { version = "2.0.0" }
 bitcoin-bech32 = "0.12"
-js-sys = "0.3.60"
+js-sys = { version = "0.3.60", optional = true }
```

Comment:
should this be removed from up here if it's specified for `wasm32` below?

Reply:
it is set as optional, not really sure how all these target dependent flags work tbh